download_data.py

```diff
@@ -44,16 +46,12 @@ def download_data(
 
 
 if __name__ == "__main__":
-    # Initialize, setup, and parse the argument parser
-    parser = ArgumentParser(prog="Data Downloader")
+    args = sys.argv
+    config_path =args[1]
 
-    parser.add_argument("--dataset-name", type=str, required=True,
-        help="Hugging Face dataset name. Should also set --dataset-subset.")
-    parser.add_argument("--dataset-subset", type=str, required=True,
-        help="Subset/config to use for Hugging Face dataset.")
-    parser.add_argument("--datasets-dir", type=str, required=True,
-        help="Path to directory in which Hugging Face datasets are downloaded.")
+    with open(config_path, 'r') as f:
+        config = yaml.safe_load(f)
 
```

Comment:
The white space here is excessive, in my opinion. This is repeated in the other files though, so it is consistent. Let's decide on if to keep all the blank lines or condense them before approving the pull request.